operator/v1/types_network.go

```diff
@@ -60,6 +60,11 @@ type NetworkSpec struct {
 	// 'false' and multiple network support is enabled.
 	DisableMultiNetwork *bool `json:"disableMultiNetwork,omitempty"`
 
+	// useMultiNetworkPolicy specifies whether or not multiNetworkPolicy
+	// should be used. If unset, this property defaults to 'false'
+	// and multiNetworkPolicy is not used.
+	UseMultiNetworkPolicy *bool `json:"useMultiNetworkPolicy,omitempty"`
```

Comment:
What is NetworkPolicy here? How this is different than above `DisableMultiNetwork`?

Reply:
How about changing the comment to:

```
UseMultiNetworkPolicy enables a controller which allows for NetworkPolicy-like objects to be used on additional networks as created by Multus CNI
```

Reply:
Changed.

Reply:
It's still not clear what's the difference between these two fields.